webhooks/virtualmachine/validation/scsi_controller.go

```diff
@@ -0,0 +1,172 @@
+// © Broadcom. All Rights Reserved.
+// The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+// SPDX-License-Identifier: Apache-2.0
+
+package validation
+
+import (
+	"fmt"
+
+	vmopv1 "github.com/vmware-tanzu/vm-operator/api/v1alpha5"
+	pkgctx "github.com/vmware-tanzu/vm-operator/pkg/context"
+	pkgutil "github.com/vmware-tanzu/vm-operator/pkg/util"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+const (
+	invalidControllerBusNumberRangeFmt     = "must be between 0 and %d"
+	invalidControllerBusNumberDoesNotExist = "SCSI controller with bus number %d does not exist"
```

Comment:
Simplify and make generic, ex.: `controller %s:%d does not exist`. That may be:

* `controller SCSI:0 does not exist`
* `controller IDE:1 does not exist`
* `controller NVME:2 does not exist`
* `controller SATA:3 does not exist`